src/SFA.DAS.EmployerAccounts/Services/AuditApiClientForSupportUser.cs

```diff
@@ -6,24 +6,24 @@
 using SFA.DAS.EmployerUsers.WebClientComponents;
 using System.IdentityModel.Claims;
 using System.Collections.Generic;
-using SFA.DAS.EmployerAccounts.Configuration;
 
 namespace SFA.DAS.EmployerAccounts.Services
 {
     public class AuditApiClientForSupportUser : IAuditApiClient
     {
         private readonly IAuditApiClient _client;
+        private readonly IUserContext _userContext;
         private readonly IAuthenticationService _authenticationService;
```

Comment:
The IAuthenticationService dependency could be removed from this class if either the Email and UserId are moved to the IUserContext interface or the GetClaimValue method is exposed by the IUserContext interface.